app/lib/dttp/access_token.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Dttp
+  class AccessToken
```

Comment:
This was a service object before. I think a better home for this code is under `app/lib/dttp`. I also refactored it it be a bit leaner. Feedback welcome.

Reply:
Happy with this. Pushing dttp related services into the `app/lib/` makes sense to me